### PR TITLE
CONTRIBUTING.md typo and error fixes

### PR DESCRIPTION
**Pull request by PluckyPrecious**
Signed-off-by: PluckyPrecious <prekel27@gmail.com>

**Description**
This helps fix typo, link  and some grammatical error in the contributing.md
This PR fixes #
An update of https://github.com/layer5io/meshery/issues/1344#issuecomment-667572034_ https://github.com/layer5io/meshery/issues/1344#issuecomment-667674809

**Notes for Reviewers**
@kushthedude kindly review, thanks.
**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: CONTRIBUTING-gitflow.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ git fetch upstream
 ```
 git branch -va
 ```
-Now, checkout your own master branch and merge the upstream repo's master branch:
+Now, check out your master branch and merge the upstream repo's master branch:
```
Comment (Jashpatel1) [inline CONTRIBUTING-gitflow.md:32]: checkout will be better as it corresponds to the command

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -81,8 +81,8 @@ __Please note__: All `make` commands should be run in a terminal from within the
 ### Prerequisites for building Meshery in your development environment:
 1. `Go` version 1.11+ installed if you want to build and/or make changes to the existing code.
 1. `GOPATH` environment variable should be configured appropriately
-1. `npm` and `node` should be installed your machine, preferably the latest versions.
-1. Fork this repository (`git clone https://github.com/layer5io/meshery.git`), clone your forked version of Meshery to your local, preferably outside `GOPATH`. If you happen to checkout Meshery inside your `GOPATH` and you have version of `Go` prior to version 1.13, please set an environment variable `GO111MODULE=on` to enable GO Modules.
+1. `npm` and `node` should be installed in your machine, preferably the latest versions.
+1. Fork this repository (`git clone https://github.com/layer5io/meshery.git`), clone your forked version of Meshery to your local, preferably outside `GOPATH`. If you happen to checkout Meshery inside your `GOPATH` and you have a version of `Go` before version 1.13, please set an environment variable `GO111MODULE=on` to enable GO Modules.
```
Comment (Jashpatel1) [inline CONTRIBUTING.md:85]: prior seems convincing

File: docs/pages/project/contributing.md [hunk 3]
```diff
@@ -100,8 +100,8 @@ __Please note__: All `make` commands should be run in a terminal from within the
 #### Prerequisites for building Meshery in your development environment:
 1. `Go` version 1.11+ installed if you want to build and/or make changes to the existing code.
 1. `GOPATH` environment variable should be configured appropriately
-1. `npm` and `node` should be installed your machine, preferably the latest versions.
-1. Fork this repository (`git clone https://github.com/layer5io/meshery.git`), clone your forked version of Meshery to your local, preferably outside `GOPATH`. If you happen to checkout Meshery inside your `GOPATH` and you have version of `Go` prior to version 1.13, please set an environment variable `GO111MODULE=on` to enable GO Modules.
+1. `npm` and `node` should be installed in your machine, preferably the latest versions.
+1. Fork this repository (`git clone https://github.com/layer5io/meshery.git`), clone your forked version of Meshery to your local, preferably outside `GOPATH`. If you happen to checkout Meshery inside your `GOPATH` and you have a version of `Go` before version 1.13, please set an environment variable `GO111MODULE=on` to enable GO Modules.
```
Comment (Jashpatel1) [inline docs/pages/project/contributing.md:104]: prior as mentioned above

File: CONTRIBUTING-gitflow.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ Just head over to the GitHub page and click the "Fork" button. It's just that si
 git clone git@github.com:USERNAME/FORKED-PROJECT.git
 ```
 Keeping Your Fork Up to Date
-While this isn't an absolutely necessary step, if you plan on doing anything more than just a tiny quick fix, you'll want to make sure you keep your fork up to date by tracking the original "upstream" repo that you forked. To do this, you'll need to add a remote:
+While this isn't a necessary step, if you plan on doing anything more than just a tiny quick fix, you'll want to make sure you keep your fork up to date by tracking the original "upstream" repo that you forked. To do this, you'll need to add a remote:
```
Comment (chandrashritii) [inline CONTRIBUTING-gitflow.md:9]:
```suggestion
While this isn't a necessary step, if you plan on doing anything more than a tiny quick fix, you'll want to make sure you keep your fork up to date by tracking the original "upstream" repo that you forked earlier. To do this, you'll need to add a remote:
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
+All set to contribute? Grab an open issue with the [help-wanted label](../../labels/help%20wanted) and jump in. Join the [Slack account](http://slack.layer5.io) and engage in conversation. Create a [new issue](/../../issues/new/choose) if needed.  All [pull requests](/../../pulls) should reference an open [issue](/../../issues). Include keywords in your pull request descriptions, as well as commit messages, to [automatically close issues in GitHub](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords).
```
Comment (chandrashritii) [inline CONTRIBUTING.md:6]:
```suggestion
All set to contribute? Grab an open issue with the [help-wanted label](../../labels/help%20wanted) and jump in. Join our [Slack channel](http://slack.layer5.io) and engage in conversation. Create a [new issue](/../../issues/new/choose) if needed.  All [pull requests](/../../pulls) should ideally reference an open [issue](/../../issues). Include keywords in your pull request descriptions, as well as commit messages, to [automatically close related issues in GitHub](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords).
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -16,13 +16,13 @@ Not sure where to start? First, see the [newcomers welcome guide](https://docs.g
...
 
-In order to contribute to Meshery, please follow the fork-and-pull request workflow described [here](./CONTRIBUTING-gitflow.md).
+To contribute to Meshery, please follow the fork-and-pull request workflow described [here](./CONTRIBUTING-gitflow.md).
 
 ## <a name="commit-signing">Signing-off on Commits (Developer Certificate of Origin)</a>
 
 To contribute to this project, you must agree to the Developer Certificate of
 Origin (DCO) for each commit you make. The DCO is a simple statement that you,
-as a contributor, have the legal right to make the contribution.
+as a contributor, have the legal right to contribute.
```
Comment (chandrashritii) [inline CONTRIBUTING.md:25]: The original wording seems more appropriate here.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -81,8 +81,8 @@ __Please note__: All `make` commands should be run in a terminal from within the
 ### Prerequisites for building Meshery in your development environment:
 1. `Go` version 1.11+ installed if you want to build and/or make changes to the existing code.
 1. `GOPATH` environment variable should be configured appropriately
-1. `npm` and `node` should be installed your machine, preferably the latest versions.
-1. Fork this repository (`git clone https://github.com/layer5io/meshery.git`), clone your forked version of Meshery to your local, preferably outside `GOPATH`. If you happen to checkout Meshery inside your `GOPATH` and you have version of `Go` prior to version 1.13, please set an environment variable `GO111MODULE=on` to enable GO Modules.
+1. `npm` and `node` should be installed in your machine, preferably the latest versions.
```
Comment (chandrashritii) [inline CONTRIBUTING.md:84]:
```suggestion
1. `npm` and `node` should be installed on your machine, preferably the latest versions.
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -97,7 +97,7 @@ After running Meshery server, you will need to select your **Cloud Provider** by
 ### Common Issues
 
 #### Invalid C headers ( Mac OS )
-When running `make run-local` on MacOS platform. Some may face error with the crypto module in Go. This is caused due to invalid C headers in Clang installed with XCode platform tools. Replacing gcc with Clang by adding `export CC=gcc` to .bashrc / .zshrc should fix the issue. More information on the issue can be found [here](https://github.com/golang/go/issues/30072)
+When running `make run-local` on the macOS platform. Some may face errors with the crypto module in Go. This is caused due to invalid C headers in Clang installed with XCode platform tools. Replacing gcc with Clang by adding `export CC=gcc` to .bashrc / .zshrc should fix the issue. More information on the issue can be found [here](https://github.com/golang/go/issues/30072)
```
Comment (chandrashritii) [inline CONTRIBUTING.md:100]:
```suggestion
When running `make run-local` on the macOS platform, some may face errors with the crypto module in Go. This is caused due to invalid C headers in Clang installed with XCode platform tools. Replacing `gcc` with Clang by adding `export CC=gcc` to .bashrc / .zshrc should fix the issue. More information on the issue can be found [here](https://github.com/golang/go/issues/30072)
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -153,7 +153,7 @@ If you want to work on the UI, it will be a good idea to use the included UI dev
 make run-ui-dev
 ```
 
-Once you have the server up and running, you will be able to access the Meshery UI at `http://localhost:3000`. One thing to note is that for the UI dev server to work, you need Meshery server running on the default port of `9081`.
+Make sure to have Meshery server configured, up and running on the default port `http://localhost:9081` before proceeding to access and work on the UI server at `http://localhost:3000`.
```
Comment (chandrashritii) [inline CONTRIBUTING.md:156]:
```suggestion
Once you have the server configured, and running successfully on the default port `http://localhost:9081`, you may proceed to access the Meshery UI at `http://localhost:3000`.
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -178,7 +178,7 @@ update /etc/hosts
 We are using ES-Lint to maintain code quality & consistency in our UI Code. To make sure your PR passes all the UI & ES-Lint Tests, please see below :
 
 - Remember to run `make run-ui-lint` & `make run-provider-ui-lint` if you are making changes in Meshery-UI & Provider-UI respectively.
-- The above commands will only fix some basic indenting rules for you, you will have to manually check your code to ensure there is no duplications, un-used variables or un-declared constants.
+- The above commands will only fix some basic indenting rules, you will have to manually check your code to ensure there are no duplications, un-used variables or un-declared constants.
```
Comment (chandrashritii) [inline CONTRIBUTING.md:181]:
```suggestion
- The above commands will only fix some basic indenting rules. You will have to manually check your code to ensure there are no duplications, un-used variables or un-declared constants.
```

File: docs/pages/project/contributing.md [hunk 11]
```diff
@@ -13,10 +13,10 @@ Not sure where to start?
 
 Follow these steps and you'll be right at home.
 
-1. See the [*Newcomers Welcome Guide*](https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit) for how, where and why to contribute.
+1. See the [*Newcomers Welcome Guide*](https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit) for how, where, and why to contribute.
```
Comment (chandrashritii) [inline docs/pages/project/contributing.md:16]:
```suggestion
1. See the [*Community Welcome Guide*](https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit) for how, where, and why to contribute.
```

File: docs/pages/project/contributing.md [hunk 12]
```diff
@@ -13,10 +13,10 @@ Not sure where to start?
...
 
 2. Sign up for a [*MeshMate*](https://layer5.io/community#meshmate) to find the perfect Mentor to help you explore the Layer5 projects and find your place in the community:
-  - **Familiarize** yourself with all the Layer5 projects (Take a look at the [Community Drive](https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA) and the [Layer5 Repository Overview](https://docs.google.com/document/d/1brtiJhdzal_O6NBZU_JQXiBff2InNtmgL_G1JgAiZtk/edit#heading=h.uwtb5xf7b5hw): Spend time understanding each of the Layer5 initiatives through high level overviews available in the community drive and in discussions with your MeshMate.
+  - **Familiarize** yourself with all the Layer5 projects (Take a look at the [Community Drive](https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA) and the [Layer5 Repository Overview](https://docs.google.com/document/d/1brtiJhdzal_O6NBZU_JQXiBff2InNtmgL_G1JgAiZtk/edit#heading=h.uwtb5xf7b5hw): Spend time understanding each of the Layer5 initiatives through high-level overviews available in the community drive and discussions with your MeshMate.
```
Comment (chandrashritii) [inline docs/pages/project/contributing.md:19]:
```suggestion
  - **Familiarize** yourself with all the Layer5 projects (Take a look at the [Community Drive](https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA) and the [Layer5 Repository Overview](https://docs.google.com/document/d/1brtiJhdzal_O6NBZU_JQXiBff2InNtmgL_G1JgAiZtk/edit#heading=h.uwtb5xf7b5hw): Spend time understanding each of the Layer5 initiatives through high-level overviews available in the community drive and through discussions with your MeshMate.
```

File: docs/pages/project/contributing.md [hunk 13]
```diff
@@ -35,13 +35,13 @@ Relevant coding style guidelines are the Go Code Review Comments and the Formatt
...
 
-In order to contribute to Meshery, please follow the fork-and-pull request workflow described [here](./git-workflow.md).
+To contribute to Meshery, please follow the fork-and-pull request workflow described [here](./CONTRIBUTING-gitflow.md).
 
 ### <a name="commit-signing">Signing-off on Commits (Developer Certificate of Origin)</a>
 
 To contribute to this project, you must agree to the Developer Certificate of
 Origin (DCO) for each commit you make. The DCO is a simple statement that you,
-as a contributor, have the legal right to make the contribution.
+as a contributor, have the legal right to contribute.
```
Comment (chandrashritii) [inline docs/pages/project/contributing.md:44]: The original wording seems more appropriate here.

File: docs/pages/project/contributing.md [hunk 14]
```diff
@@ -100,8 +100,8 @@ __Please note__: All `make` commands should be run in a terminal from within the
 #### Prerequisites for building Meshery in your development environment:
 1. `Go` version 1.11+ installed if you want to build and/or make changes to the existing code.
 1. `GOPATH` environment variable should be configured appropriately
-1. `npm` and `node` should be installed your machine, preferably the latest versions.
-1. Fork this repository (`git clone https://github.com/layer5io/meshery.git`), clone your forked version of Meshery to your local, preferably outside `GOPATH`. If you happen to checkout Meshery inside your `GOPATH` and you have version of `Go` prior to version 1.13, please set an environment variable `GO111MODULE=on` to enable GO Modules.
+1. `npm` and `node` should be installed in your machine, preferably the latest versions.
```
Comment (chandrashritii) [inline docs/pages/project/contributing.md:103]:
```suggestion
1. `npm` and `node` should be installed on your machine, preferably the latest versions.
```

**Comment by kushthedude** [comment]
Test failure is not due to the PR, can be ignored safely!

File: docs/pages/project/contributing.md (hunk 13 above)
Comment (leecalcote) [reply docs/pages/project/contributing.md:44]: Yes, this needs to stay as the original wording. The meaning changes under the new wording.

**Comment by PluckyPrecious** [comment]
Hi 👋🏽 @Jashpatel1 @chandrashritii @kushthedude @leecalcote kindly review, I've made the changes.

**Comment by kushthedude** [comment]
Great work @PluckyPrecious 🥳 !!

**Pull request merged**